apps/vr-tests/src/stories/Accordion.stories.tsx

```diff
@@ -0,0 +1,149 @@
+  AccordionHeader,
+  AccordionPanel,
+} from '@fluentui/react-accordion';
+import { CircleRingIcon } from '@fluentui/react-icons-mdl2';
+
+import { FluentProviderDecorator } from '../utilities/index';
+
+storiesOf('Accordion', module)
+  .addDecorator(story => (
+    <Screener
+      steps={new Screener.Steps()
+        .snapshot('normal', { cropTo: '.testWrapper' })
+        .focus('#opened-btn')
+        .snapshot('focus opened', { cropTo: '.testWrapper' })
+        .focus('#closed-btn')
+        .snapshot('focus closed', { cropTo: '.testWrapper' })
+        .end()}
+    >
+      {story()}
+    </Screener>
+  ))
+  .addDecorator(FluentProviderDecorator)
+  .addStory('visibility+focus', () => (
+    <Accordion index={[0]}>
+      <AccordionItem>
+        <AccordionHeader button={{ id: 'opened-btn' }}>Opened</AccordionHeader>
+        <AccordionPanel>Visible Panel</AccordionPanel>
+      </AccordionItem>
+      <AccordionItem>
+        <AccordionHeader button={{ id: 'closed-btn' }}>Closed</AccordionHeader>
+        <AccordionPanel>Hidden Panel</AccordionPanel>
+      </AccordionItem>
+    </Accordion>
+  ));
+
+storiesOf('Accordion', module)
+  .addDecorator(story => (
+    <Screener steps={new Screener.Steps().snapshot('normal', { cropTo: '.testWrapper' }).end()}>
+      {story()}
+    </Screener>
+  ))
+  .addDecorator(FluentProviderDecorator)
+  .addStory('size', () => (
+    <Accordion index={[0, 1, 2, 3]}>
+      <AccordionItem>
+        <AccordionHeader size="small">Small</AccordionHeader>
+        <AccordionPanel>Small Panel</AccordionPanel>
+      </AccordionItem>
+      <AccordionItem>
+        <AccordionHeader size="medium">Medium</AccordionHeader>
+        <AccordionPanel>Small Panel</AccordionPanel>
+      </AccordionItem>
+      <AccordionItem>
+        <AccordionHeader size="large">Large</AccordionHeader>
+        <AccordionPanel>Small Panel</AccordionPanel>
+      </AccordionItem>
+      <AccordionItem>
+        <AccordionHeader size="extra-large">Extra Large</AccordionHeader>
+        <AccordionPanel>Small Panel</AccordionPanel>
+      </AccordionItem>
+    </Accordion>
+  ))
+  .addStory('expandIconPosition="end"', () => (
+    <Accordion expandIconPosition="end" index={[0]}>
+      <AccordionItem>
+        <AccordionHeader>Opened</AccordionHeader>
+        <AccordionPanel>Visible Panel</AccordionPanel>
+      </AccordionItem>
+      <AccordionItem>
+        <AccordionHeader>Closed</AccordionHeader>
+        <AccordionPanel>Hidden Panel</AccordionPanel>
+      </AccordionItem>
+    </Accordion>
+  ))
+  .addStory('expandIcon="<Icon/>"', () => (
+    <Accordion index={[]}>
+      <AccordionItem>
+        <AccordionHeader expandIcon={<CircleRingIcon />} expandIconPosition="start">
+          Expand Icon Start
+        </AccordionHeader>
+        <AccordionPanel>Hidden Panel</AccordionPanel>
+      </AccordionItem>
+      <AccordionItem>
+        <AccordionHeader expandIcon={<CircleRingIcon />} expandIconPosition="end">

```

Comment:
![image](https://user-images.githubusercontent.com/14183168/114147913-7b7db480-9919-11eb-9eeb-1857c3ca2289.png)

⚠ It looks that this case is broken, I don't an icon on end.